app/channels/editor_channel.rb

```diff
@@ -0,0 +1,59 @@
+# The Editor Channel is opened when a user navigates to the `Edit` page of an
+# Evidence, Issue, Note, or Card. The channel is closed when the user navigates
+# away from the `edit` page. When open the channel receives updates from the
+# editor and saves them as new revisions of the resource.
+class EditorChannel < ApplicationCable::Channel
+  include ProjectScopedChannels
+  include PaperTrailActivity
+
+  attr_accessor :resource
+
+  AUTOSAVE_EVENT = 'auto-save'.freeze
+
+  def subscribed
+    reject and return unless find_resource(params)
+
+    stream_for [current_user, current_project, resource]
+  end
+
+  def save(params)
+    resource.paper_trail_event = AUTOSAVE_EVENT
+
+    if resource.update_attributes parsed_params(params['data'])
+      track_activity(resource, AUTOSAVE_EVENT)
+
+      self.class.broadcast_to([current_user, current_project, resource], resource.reload.updated_at.to_i)
```

Comment:
Can you fill me in on what we're broadcasting here?

Reply:
it's the updated_at timestamp. Each form has an `original_updated_at_time` used for conflict resolution. So every time autosave happens we broadcast back the updated_at timestamp so we can update it in the form. Then when the user hits save and conflict resolution happens it'll work as expected.

Reply:
Seems worthy of more in file comments. I'm going to update with that.